_posts/2018-11-28-how-we-work-exercise-and-stand-desk.md

```diff
@@ -0,0 +1,83 @@
+# How We Work: Exercise and Stand Desks
+
+by Cameron Gera (@gera-cameron)
+
+![A photo of ITProTV Team doing push ups][]
+
+## Why
+
+The culture of our engineering team here at ITProTV encourages healthy habits. We all enjoy taking
+care of our bodies, so why wouldn't we want to do the same at work! The two main ways we do this is
+through, you guessed it, team exercise and stand desks. There is a lot of research out there on what
+improves workplace morale, but for us, these two elements help our team be the happiest and most
+productive we can be.
+
+## Team Exercise
+
+This element of our workday occurs at the 50th of every hour between eight and five. The exercises we do vary day
+to day because we generally get sore. Some of the exercises include:
+
+- Push-ups
+- Crunches
+- Lunges
+
+I know it sounds like a lot to do it every hour, but we have seen quite a few benefits of this:
+
+1. Wakes us up and gets our blood pumping
+1. Allows a break from the code we are writing
+1. Increased productivity
+
+**Crazy Stat!** As a team, we have done over 20,000 push-ups since June 28th, 2018.
+
+This experiment has positively affected our team here. With taking just a few extra minutes a day,
+we increased our work completed by 17% for a 4-month span. We have also encouraged other ITProTV
+employees to join our Slack channel and together we keep each other accountable in living a healthy
+lifestyle.
+
+## Stand Desk
+
+For our engineering team, stand desks offer another great way to keep a healthy lifestyle. As a team,
+our use of the stand desk varies. I stand a few times a day and generally use a balance
+board, while one of my coworkers stands a majority of the day. If you've read our previous
+[post][] on
+pair programming, you know that we spend a lot of time at stand desk pairing stations, so we tend
+to agree to stand or sit for a given pairing station.
```

Comment:
The end of this sentence tripped me up. I expected it to say either "... stand or sit *at* a given pairing station." or "... stand or sit for a given pairing *rotation*."